src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/PropertyBasedEmbeddingProviderConfigStore.java

```diff
@@ -25,6 +25,10 @@ public EmbeddingProviderConfigStore.ServiceConfig getConfiguration(
         serviceName,
         serviceName,
         config.providers().get(serviceName).apiKey(),
-        config.providers().get(serviceName).url().toString());
+        config.providers().get(serviceName).url().toString(),
```

Comment:
Is there any cache of mismatch so that `config.providers().get(serviceName)` could return `null`?
If so, may want to explicitly do that part, throw an exception about "No service with name '%s'`.

Reply:
Hmm., possible if the provider is disabled after a while. Fixed it.